airbyte-integrations/connectors/source-simplesat/manifest.yaml

```diff
@@ -0,0 +1,265 @@
+version: 5.7.5
+
+type: DeclarativeSource
+
+description: |
+  This is the setup guide for the Simplesat source connector.
```

Comment:
The description is about the provider. Please provide a short paragraph explaining what this source does.